development-guidelines/workflow.md

```diff
@@ -1,43 +1,43 @@
 
-Consider special features of your contract:
+2. **Consider special features of your contract:**
 
-- [ ] Are your contracts upgradeable? Review your upgradeability code for flaws with [`slither-check-upgradeability`](https://github.com/crytic/slither/wiki/Upgradeability-Checks) or [Crytic](https://blog.trailofbits.com/2020/06/12/upgradeable-contracts-made-safer-with-crytic/). We've documented 17 ways upgrades can go sideways.
-- [ ] Do your contracts purport to conform to ERCs? Check them with [`slither-check-erc`](https://github.com/crytic/slither/wiki/ERC-Conformance). This tool instantly identifies deviations from six common specs.
-- [ ] Do you have unit tests in Truffle? Enrich them with [`slither-prop`](https://github.com/crytic/slither/wiki/Property-generation). It automatically generates a robust suite of security properties for features of ERC20 based on your specific code.
-- [ ] Do you integrate with 3rd party tokens? Review our [token integration checklist](./token_integration.md) before relying on external contracts.
+- If your contracts are upgradeable, review your upgradeability code for flaws using [`slither-check-upgradeability`](https://github.com/crytic/slither/wiki/Upgradeability-Checks) or [Crytic](https://blog.trailofbits.com/2020/06/12/upgradeable-contracts-made-safer-with-crytic/). We have documented 17 ways upgrades can go sideways.
+- If your contracts claim to conform to ERCs, check them with [`slither-check-erc`](https://github.com/crytic/slither/wiki/ERC-Conformance). This tool instantly identifies deviations from six common specs.
+- If you have unit tests in Truffle, enrich them with [`slither-prop`](https://github.com/crytic/slither/wiki/Property-generation). It automatically generates a robust suite of security properties for features of ERC20 based on your specific code.
+- If you integrate with third-party tokens, review our [token integration checklist](./token_integration.md) before relying on external contracts.
 
-Visually inspect critical security features of your code:
+3. **Visually inspect critical security features of your code:**
 
-- [ ] Review Slither's [inheritance-graph](https://github.com/trailofbits/slither/wiki/Printer-documentation#inheritance-graph) printer. Avoid inadvertent shadowing and C3 linearization issues.
-- [ ] Review Slither's [function-summary](https://github.com/trailofbits/slither/wiki/Printer-documentation#function-summary) printer. It reports function visibility and access controls.
-- [ ] Review Slither's [vars-and-auth](https://github.com/trailofbits/slither/wiki/Printer-documentation#variables-written-and-authorization) printer. It reports access controls on state variables.
+- Review Slither's [inheritance-graph](https://github.com/trailofbits/slither/wiki/Printer-documentation#inheritance-graph) printer to avoid inadvertent shadowing and C3 linearization issues.
```

Comment:
```suggestion
- [ ] Review Slither's [inheritance-graph](https://github.com/trailofbits/slither/wiki/Printer-documentation#inheritance-graph) printer to avoid inadvertent shadowing and C3 linearization issues.
```